quickwit/quickwit-config/src/source_config/mod.rs

```diff
@@ -365,6 +370,18 @@ pub struct VecSourceParams {
 #[serde(deny_unknown_fields)]
 pub struct VoidSourceParams;
 
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize, utoipa::ToSchema)]
+#[serde(deny_unknown_fields)]
+pub struct NatsSourceParams {
+    /// The subject that the source consumes.
+    pub subject: String,
```

Comment:
I was checking how people subscribe to multiple subjects ->
https://github.com/nats-io/nats-server/issues/2515
https://github.com/nats-io/nats-server/pull/3500

Reply:
I guess lets support that when it is implemented in nats

Reply:
Looks like we dont need to do anything..
But, anyways ~ do you know their branching strategy with `dev` branch(?)  probably dev is for v3?

Reply:
Ah, its for next minor version -> `2.10` probably

https://github.com/nats-io/nats-server/pull/4300/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R18-R19